esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -673,7 +674,31 @@ define([
                             node.innerText = Utility.valueCleanUp(value);
                         },
                     },
-                    Modified: { label: this.i18n.ModifiedUTCGMT, width: 162 }
+                    MinSkew: {
+                        label: nlsHPCC.MinSkew, width: 60,
+                        formatter: function (value, row) {
+                            return value !== undefined ? value : "";
+                        }
+                    },
+                    MaxSkew: {
+                        label: nlsHPCC.MaxSkew, width: 60,
+                        formatter: function (value, row) {
+                            return value !== undefined ? value : "";
+                        }
+                    },
+                    Modified: { label: this.i18n.ModifiedUTCGMT, width: 162 },
+                    AtRestCost: {
+                        label: nlsHPCC.FileCostAtRest, width: 100,
+                        formatter: function (cost, row) {
+                            return Session.formatCost(cost || 0) + " (" + (dojoConfig.currencyCode || "$") + ")";
+                        }
+                    },
+                    AccessCost: {
+                        label: nlsHPCC.FileAccessCost, width: 100,
+                        formatter: function (cost, row) {
+                            return Session.formatCost(cost || 0) + " (" + (dojoConfig.currencyCode || "$") + ")";
```

Comment:
Minor: We should start using `??` instead of `||` in this scenario.

Reply:
`" (" + (dojoConfig.currencyCode || "$") + ")";` and related formatting helpers could be moved into Session file?